src/lib/transit/planning/path-finder.ts

```
@@ -0,0 +1,242 @@
+  constructor(
+    private getKey: (node: T) => string,
+    private getNeighbors: (node: T) => T[],
+    private heuristic: (node: T, goal: T) => number,
+    private getCost: (from: T, to: T) => number,
+    private isGoal: (node: T, goal: T) => boolean
+  ) {
+    this.openSet = new PriorityQueue<T>(getKey);
+    this.closedSet = new Set<string>();
+  }
+
+  findPath(start: T, goal: T, maxIterations: number = 10000): T[] {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)